rotkehlchen/exchanges/utils.py

```diff
@@ -7,10 +7,14 @@
 
 
 def get_key_if_has_val(mapping: Dict[str, Any], key: str) -> Optional[str]:
-    """Gets the key from mapping if it exists and has a value (non empty string)"""
+    """Gets the key from mapping if it exists and has a value (non empty string)
+
+    The assumption here is that the value of the key is str. If it's not str
+    then this function will attempt to turn it into one.
+    """
     val = mapping.get(key, None)
     # empty string has falsy value
-    return val if val else None
+    return str(val) if val else None
```

Comment:
This fixes the bug for all places where this function is used (all exchange deposit/withdrawals). We force a string.